fenixedu-ist-giaf-invoices/src/main/java/pt/ist/fenixedu/giaf/invoices/ErrorLogConsumer.java

```diff
@@ -0,0 +1,22 @@
+package pt.ist.fenixedu.giaf.invoices;
+
+@FunctionalInterface
+public interface ErrorLogConsumer {
+
+    void accept(final String oid,
```

Comment:
This is a parameter nightmare, why don't you just use an object with a plain old Consumer?